subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -359,14 +359,6 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_pdu *rx)
 
 	h = (void *)p->ext_hdr_adv_data;
 
-	/* Regard PDU as invalid if a RFU field is set, we do not know the
-	 * size of this future field, hence will cause incorrect calculation of
-	 * offset to ACAD field.
-	 */
```

Comment:
As the comment states, if this validation is removed, we should add new validation to ensure offset to ACAD is current if future BT Spec adds a field in the extended header that could move ACAD around?

Reply:
Please leave your above comment as code comment, so that it is easy with `git blame` to trace back in history. If you have references to the errata please do add that too. Thank you for testing and discovering the issue with peer device setting this bit. Do we know why they set this bit?

Reply:
I added a note in place of the current one explaining why it should be safe to ignore.

>  Do we know why they set this bit?

I expect it is simply left uninitialized in their implementation by mistake; But no we do not know.